downstream/assemblies/platform/assembly-gw-dashboard.adoc

```diff
@@ -0,0 +1,18 @@
+ifdef::context[:parent-context: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+[id="gw-dashboard_{context}"]
+= {PlatformNameShort} dashboard
+
+:context: gw-dashboard
+
+[role="_abstract"]
+The {PlatformName} dashboard provides automation management and monitoring capabilities, allowing you to administer and configure automation functions, as well as view recent job activity details and performance statistics related to it. 
```

Comment:
Should this be `{PlatformNameShort}` attribute, just like the title? Ignore this comment if this is intentional.

Reply:
Good catch. This is fixed now.